spec/env/Dockerfile.centos

```diff
@@ -8,6 +8,7 @@ RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-*
 RUN sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-*
 
 RUN yum install -y ruby-devel git gcc make redhat-rpm-config
+RUN yum --enablerepo=powertools install -y libffi-devel
```

Comment:
Judging [by](https://github.com/ffi/ffi/actions/runs/10885011425/job/30202139602?pr=1119#step:4:679):

~~~
#10 1.669  libffi-devel         x86_64         3.1-22.el8            baseos          29 k
~~~

The guess was not correct. IOW the package is available in `baseos` repo which is always enabled.

Reply:
Cool, I'll retry with the package in the regular list instead, then, the one on the line previous.

Reply:
> Cool, I'll retry with the package in the regular list instead

Yep, that should work